### PR TITLE
(DOCSP-25566): Web SDK landing page refresh

### PR DESCRIPTION
**Pull request by mongodben**
## Pull Request Info

refresh to Web SDK landing page.

note: changes on top of https://github.com/mongodb/docs-realm/pull/2181 to use assets added in that PR

### Jira

- https://jira.mongodb.org/browse/DOCSP-25566

### Staged Changes (Requires MongoDB Corp SSO)

- [Web SDK (landing page)](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/DOCSP-25566/web/)

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/web.txt [hunk 1]
```diff
@@ -14,25 +18,198 @@ Realm Web SDK
...
+      Minimal-explanation code examples of how to work with the Realm Web SDK.
+
+   .. card::
+      :headline: Working Example App
+      :cta: Explore an Example App
+      :url: https://www.mongodb.com/docs/atlas/app-services/template-apps/
+      :icon: /images/icons/branding_2022/Technical_REALM_Mobile3x.png
+      :icon-alt: Realm Mobile Icon
+
+      Learn from example by dissecting a working React app that
```
Comment (mongomoe) [inline source/web.txt:63]:
```suggestion
      Learn by example through dissecting a working React app that
```
"Learn from example" sounds off

File: source/web.txt [hunk 2]
```diff
@@ -14,25 +18,198 @@ Realm Web SDK
...
+      :tabid: query-mongodb
+
+      .. procedure::
+
+         .. step:: Configure Atlas App Services
+
+            You can use the Realm Web SDK to query MongoDB Atlas data directly
+            from the browser. Perform CRUD, aggregation, and listen for database changes.
+
+            First you must :ref:`create an App Services App <create-app-ui>`,
```
Comment (mongomoe) [inline source/web.txt:87]:
```suggestion
            First, you must :ref:`create an App Services App <create-app-ui>`,
```

File: source/web.txt [hunk 3]
```diff
@@ -14,25 +18,198 @@ Realm Web SDK
...
+            For more information, see :ref:`Authenticate Users <web-authenticate>`.
+
+         .. step:: Query MongoDB
+
+            Query MongoDB directly from the browser using your authenticated user.
+            The user can only access data that they're authorized to.
+
+            For more information, see the :ref:`Query MongoDB documentation <web-query-mongodb>`.
+
+      .. image:: /images/illustrations/Spot_BrightYellow_Query_Tech_Compass.png
```
Comment (mongomoe) [inline source/web.txt:112]: Can we reduce the size of this? Looks a bit too big, it's currently part way into step 2. 
Screen Shot 2022-10-05 at 2 56 53 PM

File: source/web.txt [hunk 4]
```diff
@@ -14,25 +18,198 @@ Realm Web SDK
...
+
+         .. step:: Configure Atlas GraphQL API
+
+            Configure the GraphQL API in Atlas App Services.
+            The GraphQL API automatically generates a hosted endpoint
+            with GraphQL queries and mutations based on JSON schemas
+            for the documents in your database.
+
+            Configure App Services Authentication with data access rules to
+            control which users have access to what data.
```
Comment (mongomoe) [inline source/web.txt:128]: Question/Needed fix: Is "data. provider" a typo?

**Review by mongomoe — APPROVED** [review]
1 potential fix needed, but looks great so far

Edit: whoops accidentally clicked the approve button instead of the comment button

**Review by mongomoe — COMMENTED** [review]
1 potential fix needed, but looks great so far

File: source/web.txt (hunk 3 above)
Comment (mongodben) [reply source/web.txt:112]: cannot i don't think. stretches to fill column.

File: source/web.txt [hunk 5]
```diff
@@ -14,25 +18,198 @@ Realm Web SDK
...
+            provider credentials associated with your app.
```
Comment (mongodben) [inline source/web.txt:129]:
```suggestion
```

**Pull request merged**